src/main/java/com/iota/iri/Milestone.java

```diff
@@ -179,6 +179,10 @@ public void init(final SpongeFactory.Mode mode, final LedgerValidator ledgerVali
 
     }
 
+    public int getMilestoneStartIndex() {
```

Comment:
What do you need this method for?

Reply:
I refactored `EntryPointSelectorImpl` to use the config class in the constructor instead of copying primitive values. Now one of the values that was in the constructor was the `milestoneStartIndex`.
But we also pass `Milestone` to the constructor that carries the same information...

So this change is similar in spirit to the rest of the changes in this PR.